checkbox-ng/plainbox/impl/unit/packaging.py

```diff
@@ -153,6 +154,22 @@ def os_version_id(self):
         """Version of the operating system."""
         return self.get_record_value(self.Meta.fields.os_version_id)
 
+    @property
+    def Depends(self):
```

Comment:
Why is this is capitalized (and the following ones)?

Reply:
Good question. I didn't go too deep into this one, I just followed the previous schema to declare properties required for the new validators.
If I changed the name of the field to `depends`, I am getting this error: 
```
  File "/home/fernando/checkbox/checkbox-ng/plainbox/impl/unit/validators.py", line 582, in check_in_context
    value = getattr(unit, field2prop(field))
AttributeError: 'PackagingMetaDataUnit' object has no attribute 'Depends'. Did you mean: 'depends'?
```
Again, this is a bit confusing, but I think since we are getting the attribute based on the field name, they have to be the same. 

I tried changing the ´field2prop´ function so we translate fields starting with caps to lowercase, as we are doing with "-" and "_":
```
  def field2prop(field):
-   return str(field).replace('-', '_')
+   return str(field).replace('-', '_').lower()
```
But doing that this way created a conflict, because there is already a field named "depends" on JobDefinition units

Reply:
One solution could be to modify the "field2prop" function, translating fields starting with caps to lower, and create a new validator, that only checks the values for the same unit type:

Reply:
Thank you for your explanations.
Indeed it looks like the use of capitalized property names is dictated by the automagic of turning fields from .pxus to props.
I don't like it, but I guess we need to roll with that at least for now.